pytest_tests/tests/services/rest_gate/test_rest_headers.py

```diff
@@ -217,13 +217,12 @@ def test_negative_put_and_get_object3(
         with allure.step(
             "[Negative] Try to download (wget) object via wget with attributes [peace=peace]"
         ):
-            request = f"/get/{storage_object_1.cid}/peace/peace"
```

Comment:
But why? I thought it should be a proper request to `/get_by_attribute/` that fails because of missing container.

Reply:
You are right, inside `try_to_get_object_via_passed_request_and_expect_error` there is a call to `/get_by_attribute/` and it works as expected, but if we pass the request string directly to the method it uses it, but there is no sense in it, since we pass attritbutes there. This test just has never worked properly.